src/routes/(console)/project-[region]-[project]/overview/platforms/createAndroid.svelte

```diff
@@ -173,8 +176,8 @@ const val APPWRITE_PUBLIC_ENDPOINT = "${sdk.forProject(page.params.region, page.
                     <Typography.Text variant="m-500"
                         >2. Open the file <InlineCode
                             size="s"
-                            code="data/repository/AppwriteRepository.kt" /> and update the configuration
-                        settings.</Typography.Text>
+                            code="app/src/main/java/io/appwrite/starterkit/constants/AppwriteConfig.kt" />
```

Comment:
do we need to show a full path here?
how does it look on Console? Ideally all files are under the package.name anyways: `app/src/main/java/io/appwrite`